lib/cog_api/http/client.ex

```diff
@@ -134,20 +134,36 @@ defmodule CogApi.HTTP.Client do
     RelayGroups.delete(relay_group_id, endpoint)
   end
 
-  def relay_group_add_relay(relay_group_id, relay_id, %Endpoint{}=endpoint) do
-    RelayGroups.update_memberships(:add, relay_group_id, relay_id, endpoint)
+  def relay_group_add_relays_by_name(relay_group_name, relay_names, %Endpoint{}=endpoint) do
```

Comment:
So while have the function name denotes the expected inputs and that seems pretty clear what should be expected, I'm not sure that this is the pattern that we want to move forward with. Is there a problem with having a single function with multiple function heads that can take a list of ids or a list of names? This way we can call `relay_group_add_relays` and depending on the data sent, we can act accordingly.